bin/desi_run_night

```diff
@@ -23,6 +23,8 @@ def parse_args():  # options=None):
                         help="The queue to submit jobs to. Default is realtime.")
     parser.add_argument("-r", "--reservation", type=str, required=False, default=None,
                         help="The reservation to submit jobs to. If None, it is not submitted to a reservation.")
+    parser.add_argument("--system-name", type=str, required=False, default=None,
```

Comment:
Can we add a default here of cori-haswell?

Reply:
In my review I missed the fact that None is set to an auto-detected default by the new `batch.get_config()` via a call to `batch.default_system()`. That works for my needs so I'll retract this comment

Reply:
Default of `None` will propagate through to `desispec.workflow.batch.default_system()` which defaults of 'cori-haswell' if on cori, 'dirac' if on hpcs/dirac, and 'perlmutter-gpu' if on perlmutter (guessing that NERSC will set $NERSC_HOST=perlmutter...).  We chatted offline and agreed this is ok.